src/forestTypes/basicForests/rerf/unprocessedRerFNode.h

```diff
@@ -56,50 +56,97 @@ namespace fp{
 				inline void pickMTRY(){
 					int rndMtry;
 					int rndFeature;
-					for (int i=0; i < fpSingleton::getSingleton().returnMtry(); ++i){
+					int mtryDensity = (int)((double)fpSingleton::getSingleton().returnMtry()*fpSingleton::getSingleton().returnMtryMult());
+					for (int i=0; i < mtryDensity; ++i){
 						rndMtry = std::rand() % fpSingleton::getSingleton().returnMtry();
```

Comment:
Should we be using the faster MWC here? and also for seeding...

Reply:
Yes all of the randomness should be done using MWC except for an initial seed when the user has not provided one.  The rerf forest will go away eventually though so I never made this change.